c7n/filters/iamaccess.py

```diff
@@ -251,6 +275,12 @@ def get_accounts(self):
         accounts.add(owner_id)
         return accounts
 
+    def get_vpcs(self):
```

Comment:
Are we going to need to support getting values from a file, like with whitelist_from?